spec/spec_helper.rb

```diff
@@ -145,6 +145,8 @@ def fixture(filename)
   File.join(__dir__, "fixtures", filename)
 end
 
+# This does not work. We now use CLusterable::Holding.count for scrub_runner_spec.rb
+# For commitments and ht_items we recommend a `count` method on the appropriate class.
```

Comment:
Can we just remove `cluster_count`?

Reply:
We decided to leave it as a breadcrumb since other `xit` and/or `xdescribe` tests use it and it's out of scope to go a-hunting around those use cases.